.circleci/config.yml

```diff
@@ -273,17 +273,22 @@ jobs:
       - ngrok-install
       - server-build
       - server-start
-      - app-detox-build:
-          configuration: android.emu.release
       - android/create-avd:
           avd-name: TwilioVoiceReactNativeReferenceApp_AVD
           install: true
-          system-image: system-images;android-29;default;x86_64
+          system-image: system-images;android-29;google_apis;x86_64
       - android/start-emulator:
           avd-name: TwilioVoiceReactNativeReferenceApp_AVD
+          memory: 8192
           post-emulator-launch-assemble-command: ""
           restore-gradle-cache-post-emulator-launch: false
+      - app-detox-build:
+          configuration: android.emu.release
       - app-expose-port
+      - run:
+          name: "Wait for emulator to settle"
+          command: |
+            sleep 300
```

Comment:
Are these all needed to fix the build? What is the cost difference?

Reply:
It's been helping, it seems to alleviate some flakiness I've noticed. But there's no definitive answer, just anecdotal evidence. In the end, the build will take some time anyways, more than 20 minutes.

Reply:
weird, my comment is for the whole diff in config.yml lol
not sure why it only shows up for this line. anyway, is the move to 2xlarge and increasing the memory necessary?

Reply:
It helps quite a bit. With the new 5 minute wait to let the new Google Services APIs settle, it ends up being about 15 minutes of build I noticed. When I was debugging the Android logs, I also noticed the emulator running out of RAM sometimes and crashing. I'm not sure this is the "minimum" amount of resources we need however, I just cranked it to max.